test/concurrency/mvcc_test.cpp

```diff
@@ -30,141 +30,6 @@ static std::vector<ConcurrencyType> TEST_TYPES = {
-                EXPECT_TRUE(
-                    next_tile_group_header->GetNextItemPointer(
-                                                next_location.offset).IsNull())
-                    << "Invalid version in a version chain and is not delete";
-              }
-
-              cid_t next_begin_cid = next_tile_group_header->GetBeginCommitId(
-                  next_location.offset);
-              cid_t next_end_cid =
-                  next_tile_group_header->GetEndCommitId(next_location.offset);
-
-              // 3. Timestamp consistence
-              if (next_begin_cid == MAX_CID) {
-                // It must be an aborted version, it should be at the end of the
-                // chain
-                EXPECT_TRUE(
-                    next_tile_group_header->GetNextItemPointer(
-                                                next_location.offset).IsNull())
-                    << "Version with MAX_CID begin cid is not version tail";
-              } else {
-                EXPECT_EQ(prev_end_cid, next_begin_cid)
-                    << "Prev end commit id should equal net begin commit id";
-                ItemPointer next_prev_location =
-                    next_tile_group_header->GetPrevItemPointer(
-                        next_location.offset);
-
-                // 4. Version doubly linked list consistency
-                EXPECT_TRUE(next_prev_location.offset == prev_location.offset &&
-                            next_prev_location.block == prev_location.block)
-                    << "Next version's prev version does not match";
-              }
-
-              prev_location = next_location;
-              prev_end_cid = next_end_cid;
-              next_location = next_tile_group_header->GetNextItemPointer(
-                  next_location.offset);
-            }
-
-            // Now prev_location is at the tail of the version chain
-            ItemPointer last_location = prev_location;
-            auto last_tile_group =
-                catalog_manager.GetTileGroup(last_location.block);
-            auto last_tile_group_header = last_tile_group->GetHeader();
-            // txn_id_t last_txn_id =
-            // last_tile_group_header->GetTransactionId(last_location.offset);
-            cid_t last_end_cid =
-                last_tile_group_header->GetEndCommitId(last_location.offset);
-            EXPECT_TRUE(
-                last_tile_group_header->GetNextItemPointer(last_location.offset)
-                    .IsNull())
-                << "Last version has a next pointer";
-
-            EXPECT_EQ(last_end_cid, MAX_CID)
-                << "Last version doesn't end with MAX_CID";
-          }
-        }
-      } else {
-        EXPECT_TRUE(tile_group_header->GetNextItemPointer(tuple_slot).IsNull())
-            << "Invalid tuple must not have next item pointer";
-      }
-    }
-    LOG_INFO("[OK] oldest-to-newest version chain validated");
-  }
-}
 
 TEST_F(MVCCTest, SingleThreadVersionChainTest) {
```

Comment:
This test should be enhanced in the future. originally this test case is designed for checking the correctness of version chain with oldest-to-newest ordering. However, we have changed the ordering to newest-to-oldest, so new test cases should be added in the future.